submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionApiTest.kt

```diff
@@ -190,6 +192,38 @@ class SubmissionApiTest(
         assertThat(File("$ftpPath/${submitted.relPath}/Files/folder1")).isEmptyDirectory()
     }
 
+    @Nested
+    @SpringBootTest(webEnvironment = RANDOM_PORT, properties = ["app.subBasePath=subRelPath"])
```

Comment:
this name make everything more confusing naming the base path `subRelPath` just confuse reader.  `base/path` or `submissions-base-path`